middleware/src/middleware_test.go

```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var disable = rpcmessages.ToggleSettingArgs{ToggleSetting: false}
+var enable = rpcmessages.ToggleSettingArgs{ToggleSetting: true}
```

Comment:
Wondering if, as alternative to the globals in the tests, a function similar to the one below would be better.   

```go
func getToggleSettingArgs(enabled bool) rpcmessages.ToggleSettingArgs {
	return rpcmessages.ToggleSettingArgs{ToggleSetting: enabled}
}

// would be
testMiddleware.EnableTor(getToggleSettingArgs(true))

```

Reply:
I thought about this as well, lets go with it and add back the globals check.

Reply:
Done